e2e/core/ui/CookieManager.js

```diff
@@ -1,6 +1,7 @@
 /**
  * this class handle the cookies.
  */
+//TODO remove class
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)